electron/main.ts

```diff
@@ -147,6 +148,14 @@ async function createWindow(): Promise<BrowserWindow> {
     }
   })
 
+  if (isRunningInSteam) {
+    logInfo(
+      'Heroic started via Steam. Switching to fullscreen',
+      LogPrefix.Backend
+    )
+    mainWindow.maximize()
```

Comment:
I wonder if fullscreen makes more sense than maximize https://www.electronjs.org/docs/latest/api/browser-window#winsetfullscreenflag

in a desktop it hides the taskbar and window decoration (steam big picture style), no idea how that works on the steamdeck though

Reply:
That's a good point, maybe @Nocccer could test the difference on the deck.

Reply:
Pretty hard to test without a dongle on steam deck. I will instead test big picture mode.